tmt/steps/provision/__init__.py

```diff
@@ -1702,6 +1741,36 @@ def essential_requires(cls) -> list['tmt.base.Dependency']:
 
         return []
 
+    @property
+    def logdir(self) -> Optional[Path]:
+        """
+        Path to store logs
+        """
+        return self.workdir / 'logs' if self.workdir else None
+
+    def fetch_logs(
+        self,
+        logger: tmt.log.Logger,
+        dirpath: Optional[Path] = None,
+        guest_logs: Optional[list[GuestLog]] = None,
+    ) -> None:
+        """
+        Get log content and save it to a directory.
+
+        :param logger: logger to use for logging.
+        :param dirpath: a directory to save into. If not set, (:py:attr:`logdir`),
+            or current working directory will be used.
+        :param guest_logs: list of GuestLog need to be handled. If not set, all guest logs
+            would be collected, as reported by (:py:attr:`guest_logs`).
```

Comment:
What about something like this?
```suggestion
        :param dirpath: a directory to save into. If not set, :py:attr:`logdir`,
            or current working directory will be used.
        :param guest_logs: optional list of :py:attr:`GuestLog`. If not set,
            all guest logs from :py:attr:`Guest.guest_logs` would be collected.
```

Reply:
Yeah, better: https://github.com/teemtee/tmt/pull/3602/commits/8ff5a8e74e7168b564159d5b371788f0348254b6